fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -74,7 +74,7 @@ let modifySchedule (fn : CanvasID -> string -> Task<unit>) =
 let fns : List<BuiltInFn> =
   [ { name = fn "DarkInternal" "endUsers" 0
       parameters = []
-      returnType = TList varA
+      returnType = TList TStr
```

Comment:
When updating internal fns, should I not worry about deprecating the old version? Or is a bit of a gray area?